frontend/src/pages/EditStudent.vue

```diff
@@ -109,82 +111,68 @@ export default {
           },
           studentData: {},
           languages: [],
-          API_nativeLanguage: []
-
       }
   },
-  computed: {
-    // If NativeLanguage is changed, we assign it this value 
+  computed: {    
+    ...mapGetters(['API_nativeLanguage', 'getStudentByStudentId']),
+    ...mapState(['updateStudentSuccess']),
     nativeLanguage(){
       return this.selected ? this.selected.NativeLanguage: ''
-    }
+    },
+
   },
   watch: {
     file (val){
       this.uploadFile() 
+    },
+    updateStudentSuccess(value){
+      if(value === true){
+        this.$buefy.notification.open({
+              message: 'Student saved. <u>View profile</u>!',
+              duration: 3000,
+              type: 'is-success',
+              position: 'is-top',
+            })
+        // Refresh NativeLanguage
+        this.getNativeLanguages()
+        setTimeout(() => {
+          this.$router.push(
+            {path: `/students/${this.studentData.StudentID}`})}, 3000)
+      } 
+      else {
+         this.$buefy.notification.open({ 
+              message: 'Something went wrong. Please try again.',
+              duration: 3000, 
+              type: 'is-warning',
+              position: 'is-top'
+            })
+      }
     }
```

Comment:
```suggestion
    watchUpdateStudentSuccess(newValue, oldValue){}
```

Renamed `updateStudentSuccess()` to `watchUpdateStudentSuccess()`, same as computed method
Moved notification display into `saveStudent()` method